ompi/mca/coll/acoll/coll_acoll_module.c

```diff
@@ -132,6 +132,17 @@ mca_coll_base_module_t *mca_coll_acoll_comm_query(struct ompi_communicator_t *co
         break;
     }
 
+    // Check SMSC availability (currently only for XPMEM)
+    if (!mca_smsc_base_has_feature(MCA_SMSC_FEATURE_CAN_MAP)) {
+        opal_output_verbose(MCA_BASE_VERBOSE_ERROR, ompi_coll_base_framework.framework_output,
+                            "coll:acoll: Error: SMSC's xpmem component is not available. "
```

Comment:
nit: 
I would prefer to say here that the MAP feature is not available in the smsc component, instead of saying xpmem is not available. The accelerator smsc component actually also supports the MAP feature, but it only advertises it if the fall-back component also supports the MAP feature (for host memory). There might be a way on how to make this work however.